modules/gltf/src/lib/extensions/EXT_structural_metadata.ts

```diff
@@ -391,59 +387,110 @@ function getPropertyDataFromBinarySource(
 }
 
 /**
- * Gets offset array from `arrayOffsets` or `stringOffsets`.
- * @param {GLTFScenegraph} scenegraph - Instance of the class for structured access to GLTF data.
- * @param {number} offsets - Buffer view index
- * @param offsetType - The type of values in `arrayOffsets` or `stringOffsets`.
- * @param {number} numberOfElements - The number of elements in each property array.
- * @returns {TypedArray}
+ * Parse propertyTable.property.arrayOffsets
```

Comment:
Maybe say something about what these array offsets are for?

Reply:
Done